### PR TITLE
feat: get data from modelscope and huggingface

### PR DESCRIPTION
**Pull request by 0xff-dev**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it
feat: get data from modelscope and huggingface

#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

File: apiserver/service/forward.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+		instance = forwardrepo.NewHuggingFace(forwardrepo.WithHFToken(token), forwardrepo.WithModelID(modelID), forwardrepo.WithTransport(tp))
+	case modelscopeRepo:
+		instance = forwardrepo.NewModelScope(forwardrepo.WithHFToken(token), forwardrepo.WithModelID(modelID))
+	default:
+		ctx.AbortWithStatusJSON(http.StatusBadRequest, gin.H{
+			"message": fmt.Sprintf("Unsupported repository type %s", repo),
+		})
+		return
+	}
+	revisions, err = instance.Revisions(ctx.Request.Context())
```
Comment (nkwangleiGIT) [inline apiserver/service/forward.go:129]: the code to get the instance is duplicated for Summary and Revisions, let's refine it.

File: apiserver/service/router.go [hunk 2]
```diff
@@ -68,6 +68,9 @@ func NewServerAndRun(conf config.ServerConfig) {
 	chatGroup := r.Group("/chat")
 	registerChat(chatGroup, conf)
 
+	fg := r.Group("/forward")
+	registerForward(fg)
```
Comment (nkwangleiGIT) [inline apiserver/service/router.go:72]: do we have any auth check(authentication or authorization) for this router?

Comment (0xff-dev) [reply apiserver/service/router.go:72]: I don’t think it’s necessary, we’re just forwarding it.

Comment (bjwswang) [reply apiserver/service/router.go]: we can only check whether the user have valid platform token,then we can track the api call .

**Review by bjwswang — APPROVED** [review]

**Pull request merged**